src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -76,8 +71,101 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
         approved: true
       }
     });
-  }
+    const wbs = await prisma.wBS_Element.findUnique({
+      where: {
+        wbsElementId: foundCR.wbsElementId
+      },
+      include: {
+        workPackage: true,
+        project: true
+      }
+    });
+
+    if (!wbs) {
+      return res.status(404).json({ message: `wbs element with id #${foundCR.wbsElementId} not found` });
+    }
+
+    const wp = { ...wbs.workPackage };
+
+    const proj = { ...wbs.project };
+
+    if (!wp && proj) {
+      let updatedBudgetImpact;
+      if (!proj.budget) {
+        updatedBudgetImpact = 0 + foundPs.budgetImpact;
+      } else {
+        updatedBudgetImpact = proj.budget + foundPs.budgetImpact;
+      }
+      const change = {
+        changeRequestId: crId,
+        implementerId: reviewerId,
+        detail: buildChangeDetail('Budget Impact', String(proj.budget), String(updatedBudgetImpact))
+      };
+      await prisma.project.update({
+        where: { projectId: wbs.projectNumber },
+        data: {
+          budget: updatedBudgetImpact,
+          wbsElement: {
+            update: {
+              changes: {
+                createMany: {
```

Comment:
doesn't need to be createmany since it's just one, you can just do create

Reply:
It throws an error when I make it just create. Maybe because changes is an array.

Reply:
yeah if it's just create it doesn't need to be an array, you would do like create: { change } or maybe `create: { data: { ...change}}` or something along those lines